FrontBundle/Controller/NodeController.php

```diff
@@ -97,6 +95,19 @@ public function previewAction(Request $request)
     }
 
     /**
+     * Render the node version given by encoded $token for an alias including a language prefix
+     *
+     * @param string  $token
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function previewWithPrefixAction($token, Request $request)
```

Comment:
This was to let the opportunity to get the $languagePrefix if required. But as it's not used for now I will remove it and use a single controller.